plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcPageSinkProvider.java

```diff
@@ -38,12 +39,24 @@ public JdbcPageSinkProvider(JdbcClient jdbcClient)
     @Override
     public ConnectorPageSink createPageSink(ConnectorTransactionHandle transactionHandle, ConnectorSession session, ConnectorOutputTableHandle tableHandle)
     {
-        return new JdbcPageSink(session, (JdbcOutputTableHandle) tableHandle, jdbcClient);
+        throw new IllegalArgumentException("createPageSink not supported without TaskId");
```

Comment:
`without TaskId` is no longer accurate

Reply:
Also this is kinda annoying that you need to override those effectively deprecated methods. Maybe provide a default implementation in an interface which throws an exception?
@findepi wdyt?